src/components/locations/LocationEditDialog.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, Dialog, DialogContent, DialogTitle } from '@mui/material';
+
+import { useAppDispatch } from '@respond/lib/client/store';
+import { LocationActions } from '@respond/lib/client/store/locations';
+import { createNewTemporaryLocation, Location } from '@respond/types/location';
+
+import { LocationEditForm } from './LocationEditForm';
+
+export function LocationEditDialog({ location = createNewTemporaryLocation(), open, onSubmit, onClose }: { location?: Location; open: boolean; onSubmit: (location: Location) => void; onClose: () => void }) {
+  const dispatch = useAppDispatch();
+  const handleSubmit = (location: Location) => {
+    if (location.active) {
```

Comment:
Something more-robust will be needed, here, to handle use cases for editing a location for this activity only.